android/expoview/src/main/java/host/exp/exponent/notifications/channels/ScopedGroupSerializer.java

```diff
@@ -0,0 +1,23 @@
+package host.exp.exponent.notifications.channels;
+
+import android.app.NotificationChannelGroup;
+import android.os.Build;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
+import expo.modules.notifications.notifications.channels.serializers.ExpoNotificationsChannelGroupSerializer;
+import expo.modules.notifications.notifications.channels.serializers.NotificationsChannelSerializer;
+
+public class ScopedGroupSerializer extends ExpoNotificationsChannelGroupSerializer {
```

Comment:
Once again, consider changing inheritance into composition. You could just create two classes that maps channel ids - one for scoped and one for unscoped, and provide them here via constructor.

Reply:
see https://github.com/expo/expo/pull/9137#discussion_r453545807